WordPress/Classes/ViewRelated/Me/AccountSettingsViewController.swift

```diff
@@ -113,33 +93,16 @@ private struct AccountSettingsController: SettingsController {
                         email,
                         primarySite,
                         webAddress
-                    ]),
-                ImmuTableSection(
-                    headerText: mediaHeader,
-                    rows: [
-                        uploadSize,
-                        mediaRemoveLocation
-                    ],
-                    footerText: nil),
-                ImmuTableSection(
-                    headerText: editorHeader,
-                    rows: [
-                        visualEditor
-                    ],
-                    footerText: nil)
+                    ])
                 ])
         } else {
+            let loading = TextRow(
+                title: NSLocalizedString("Loading Settings...", comment: "Loading Settings label"),
+                value: "")
```

Comment:
That's not a great UI and not consistent with other places. If you look at "My Profile", it shows the labels with empty values while loading, which was the design decision. If that has changed, I'd take it to a separate PR.

If we're moving app settings to a separate section, then `service` should stop being an optional in this controller.

Also, you need to move the `.AccountSettings` feature check to `MeViewController` and don't show the entry if it's disabled.